.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ commands:
       - run:
           name: Run Tests
           command: |
-            cd drupal/web && php core/scripts/run-tests.sh --verbose --sqlite --module elasticsearch_helper
+            cd drupal/web && php core/scripts/run-tests.sh --suppress-deprecations --verbose --sqlite --module elasticsearch_helper
```

Comment:
Should we suppress deprecation notices? How do we see now if we are using some mappings, settings or commands that are deprecated (and dropped from next or following versions) and needs to be taken into account in the middle layer service.

Now as notices are deprecated test-es-7 part is green again even though type name parameter is defined (while creating the index) but `include_type_name` parameter is not set `true` anywhere.

It is true that Elasticsearch-PHP library is enable to index documents (despite of deprecations) but this way we do not notify those breaking changes listed in Elasticsearch-PHP library:

```
From BREAKING_CHANGES.md

# 7.4

- Using a deprecated parameter is notified triggering a [E_USER_DEPRECATED](https://www.php.net/manual/en/errorfunc.constants.php)
  error (e.g. using the `type` parameter will generate a `Specifying types in urls has been deprecated`
  deprecation message).
- When `delete` with an empty `id` an `Elasticsearch\Common\Exceptions\RuntimeException\Missing404Exception`
  exception is thrown. Previously it was a `Elasticsearch\Common\Exceptions\RuntimeException\InvalidArgumentException`.
```

Reply:
Good point. I revert that. Let's fix the deprecation notices when this is merged.